src/components/Form/InputField.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import './formFields.css';
+
+type InputProps = {
+  type: 'text' | 'date' | 'file';
+  label: string;
+  value: string;
```

Comment:
value is allways string? What if input is DatePicker? Value should be date, and you should work with dates